install.sh

```diff
@@ -444,6 +444,15 @@ download_hash() {
     HASH_EXPECTED=${HASH_EXPECTED%%[[:blank:]]*}
 }
 
+# --- check if it is a number type  ---
+isNumber() {
+    if echo "${1}" | grep -Pq '^[0-9]+([.][0-9]+)?$'; then
+        return
+    else
+        return 1
+    fi
+}
+
```

Comment:
Don't use grep, we can use sh builtin glob matching
```suggestion
# --- check if it is a number type  ---
is_number() {
  case $1 in
    ''|*[!0-9]*) return 1 ;;
    *) return 0 ;;
  esac
}

```

Reply:
I admit the shell glob I posted above was wrong but I suspect this version won't work as expected for `VERSION_ID` containing a decimal point which is quite common:

https://forums.centos.org/viewtopic.php?t=75942
https://gist.github.com/SwooshyCueb/4905ac15e3b5658f6c6907d9562af479